docker/compose.yaml

```diff
@@ -0,0 +1,19 @@
+services:
+  web:
+    build:
+      context: ../
+      dockerfile: docker/Dockerfile
+    command:
+      - /bin/bash
+      - -c
+      - |
+        npm install
+        npm start
+    environment:
+      - RESOURCE_CONFIG=configs/resources/sample-resource.yml
+      - OPERATION_CONFIG=${OPERATION_CONFIG}
```

Comment:
This might not be able to run by itself because people needs to setup their resource, instead of using the sample resource which point to a specific org now.